FH-3293: adds basic CLI arg functionality

damienomurchu opened the pull request:
Adds basic CLI argument functionality to base data-miner codebase 

To test: clone/ pull copy of data-miner to local machine, navigate to data-miner directory and test the commandline arg commands specified in the README.md file in repo

@austincunningham let me know what you think

damienomurchu committed:
adds basic CLI arg functionality

austincunningham commented on README.md:
```diff
@@ -3,8 +3,21 @@ data-miner is a data analysis application written in node.js that is called and
+* `node ./app --x=2 --y=5`
+* `node ./app -x 2 -y 5`
+* `node ./app --x 2 --y 5`
+
```
Nice to see the return from the input

austincunningham reviewed (commented):
Nice to see a  return form the input

Commit:
expands & refactors CLI func

removes redundant code

Commit:
updates readme & bumps version

updates readme with example

updates readme with cli screenshot

tidies up readme
